.github/workflows/continuous-integration-workflow.yml

```diff
@@ -139,14 +139,21 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       is_publishable_branch: ${{ steps.check-branch.outputs.is_publishable_branch }}
+      branch_tag: ${{ steps.check-branch.outputs.branch_tag }}
     steps:
       - name: check branch ${{ github.ref }} is either main or a version number
         id: check-branch
         run: |
-          if [[ ${{ github.ref }} == 'refs/heads/main' || ${{ github.ref }} =~ refs\/heads\/[0-9]+\.[0-9]+ ]]; then            
-            echo "is_publishable_branch=true" >> $GITHUB_OUTPUT
+          if [[ ${{ github.ref }} == 'refs/heads/main' ]]; then
+            echo "is_publishable_branch=true" >> "${GITHUB_OUTPUT}"
+            echo "branch_tag=latest" >> "${GITHUB_OUTPUT}"
```

Comment:
Would we want `latest` pointing at the tip of main? We consider it unstable.
Would we be better having main be `main` or `bleeding_edge` or `unstable` or `testing` or something, and work out how to match the latest release with `latest`?

Reply:
It's now "development".